### PR TITLE
Revises versioning for USEEIO models to use aliases

### PR DESCRIPTION
**Pull request by WesIngwersen**
- edit versioning scheme
 - add py script to select alias
 - add yamls for two lists of names
 -

### CODE REVIEW

**Comment by WesIngwersen** [comment]
I plan two more files at least ... a models.csv and then models.md which displays contents in md of the models.csv and aliases.csv files with some text.  The models.csv will be a master list of all USEEIO models that we've created.

File: versioning/VersioningScheme.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+National and state models have separate versioning. 
+The combination of the minor and major version reflect the model components that are included or their derivation.
+
+## Alias
+Each unique set of model data components and model attributes, not including the location or year, has an alias.
+The alias names are drawn from name lists that capture some aspect of [americana](https://en.wikipedia.org/wiki/Americana_(culture)) or the natural environment of the U.S.
+The aliases are chosen with a random script given a common set of names used for a generation and geo-resolution of models. 
+For example v1 state models aliases are drawn from names of American pies; v2 national models aliases are drawn from names of U.S. songbirds that migrate in the winter.
+
+## Base IO Schema
```
[the remark this answers is not in the document]
Comment (WesIngwersen) [reply versioning/VersioningScheme.md:42]: I agree i'm not sure about leaving it later...we might consider moving that out once aliases are further fleshed out but i left it for now.

File: versioning/VersioningScheme.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+They are not full model descriptions.
+A named model does not imply that the model has been reviewed or released. 
+Models named and created prior to the advent of this versioning scheme are not renamed. 
+
+## Naming pattern
+Model names are composed of parts in a clear pattern.
+The naming pattern is a set sequence of these parts with separator characters that are required when the given part is present.
+```
+{loc}{root} v{major}.{minor}.{patch}-{alias}-{YY}
+```
```
[the remark this answers is not in the document]
Comment (WesIngwersen) [reply versioning/VersioningScheme.md:16]: When writing the model name out we wanted to have the space like we did for USEEIO v2. I think its worth adding a note about

**Comment by WesIngwersen** [comment]
> I plan two more files at least ... a models.csv and then models.md which displays contents in md of the models.csv and aliases.csv files with some text. The models.csv will be a master list of all USEEIO models that we've created.

will save models.csv for a separate issue since this issue is really just intended to resolve the way models are named and not provide a list of all models

**Comment by bl-young** [comment]
ok i think looks good to me. script worked fine

**Pull request merged**